tests/urls.py

```diff
@@ -0,0 +1,5 @@
+# from django.conf.urls import url
+
+urlpatterns = [
+    #  url(r'^admin/', admin.site.urls),
+]
```

Comment:
Ah, I see this is referenced in the settings.. 
Can the commented lines be  cleaned up?